meson.build

```diff
@@ -383,6 +383,12 @@ if cc.has_header('linux/ipmi.h')
 else
   have_linux_ipmi = false
 endif
+if cc.has_header('linux/gpio.h')
```

Comment:
Should only be when plugin _gpio is set right?

Reply:
That's only setting the `have_linux_gpio` variable, the logic we moved into the plugins themselves some time ago.